### DISCUSSION TITLE
Can I Start to learn react with some knowledge of Javascript

### DISCUSSION THREAD

**Discussion by evansso** [community/community#60474] [Other Feature Feedback, Questions, & Ideas]
as i continue learning javascript with the knowlege of ( syntax, conditionals, functions, and scope) can i also start learning react. I find it boring as I start studying javascript only as I want to know react and build web apps

**Comment by evansso** [comment]
i want to leaarn react nd also javascript.what should i do for now . at javascript i have learned conditionals scope functions data types variables. what advice can you give me

**Comment by MUK-Dev** [comment]
Yes, you can.

I suggest learning about [es6 standards](https://www.w3schools.com/js/js_es6.asp) to JavaScript 2022 standards. These will really help you with React. 
After these, you should learn React and start small by creating projects.
An excellent free crash course is from [Academind](https://www.youtube.com/watch?v=4baq00tHfmA)
